docusaurus/docs/api/cli.md

```diff
@@ -0,0 +1,373 @@
+
+#### Usage
+Once this has been confirmed all we have to do is run the following.
+```bash
+yarn run pixeloven pretty scss
+```
+We can also target specific files.
+```
+yarn run pixeloven pretty ts ./src/index.scss
+```
+StyleLint options can be passed through the CLI but it is highly recommended that this behavior be managed with in a **stylelint.json** file.
+
+### TypeScript
+> It is important to note that currently this feature expects a **tslint.json** file to be present at the root of the package to configure the compiler.
+
+|Argument|Description|
+|---|---|
+|`ts`| A proxy for the TSLint linter with a few opinionated defaults for easier setup. |
+|`tsx`| An alias for `ts`. Implemented to help differentiate JSX based projects.|
+
+#### Options
+For reference on all the available options please review the [official documentation](https://prettier.io/) for Prettier. 
+
+#### Setup
+First be sure to create a **tslint.json** file at the root of the project or in other words adjacent to the **package.json**.
+```json
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:latest",
+    "tslint-config-prettier",
+    "tslint-eslint-rules"
+  ]
+}
+```
+PixelOven by default provides [tslint-config-prettier](https://github.com/prettier/tslint-config-prettier) for compatibility between TSLint and Prettier as well as [tslint-eslint-rules](https://github.com/buzinas/tslint-eslint-rules) to match rules against ESLint.
+
+#### Usage
+Once this has been confirmed all we have to do is run the following.
+```bash
+yarn run pixeloven pretty ts
+```
+We can also target specific files.
+```
+yarn run pixeloven pretty ts ./src/index.ts
+```
+TSLint options can be passed through the CLI but it is highly recommended that this behavior be managed with in a **tslint.json** file.
+
+## CLI: PixelOven.Test
+> NOTICE: This feature will be removed from the core cli and become an addon in a future update.
+
+Test is a simple wrapper for testing JavaScript applications. It currently acts as proxy for the Jest compiler and comes with a dependecies to make it more compatible with TypeScript.
+
+### Jest
+> It is important to note that currently this feature expects a **jest.json** file to be present at the root of the package to configure the compiler.
+
+For reference on all the available configurations please review the [official documentation](https://jestjs.io/) for Jest. 
+
+#### Setup
+First be sure to create a **jest.json** file at the root of the project or in other words adjacent to the **package.json**.
+```json
+{
+    "bail": true,
+    "collectCoverageFrom": [
+        "!<rootDir>/**/*.stories.{js,jsx,ts,tsx}",
+        "!<rootDir>/**/*.test.{js,jsx,ts,tsx}",
+        "<rootDir>/**/*.{js,jsx,ts,tsx}"
+    ],
+    "coverageDirectory": "<rootDir>/../coverage",
+    "coverageThreshold": {
+        "global": {
+            "branches": 0,
```

Comment:
Can we make these 90 to set a standard to shoot for and let folks downgrade as needed?